packages/css-framework/package.json

```diff
@@ -1,41 +1,49 @@
 {
   "name": "@royalnavy/css-framework",
-  "description": "Foundational CSS Framework for Royal Navy components and applications.",
   "version": "2.14.0-alpha.2",
+  "main": "index.scss",
+  "author": "Royal Navy",
+  "license": "Apache-2.0",
   "publishConfig": {
     "access": "public"
   },
-  "author": "Royal Navy",
-  "contributors": [
-    "hxltrhuxze <hxltrhuxze@royalnavy.io> (royalnavy.io)",
-    "antictor <antictor@royalnavy.io> (royalnavy.io)"
+  "files": [
+    "dist",
+    "src"
   ],
   "repository": {
     "type": "git",
     "url": "https://github.com/Royal-Navy/design-system.git",
     "directory": "packages/css-framework"
   },
-  "homepage": "https://github.com/Royal-Navy/design-system/tree/develop/packages/css-framework#readme",
-  "license": "MIT",
-  "main": "index.scss",
-  "files": [
-    "dist",
-    "src"
-  ],
   "scripts": {
-    "build": "sass --style=compressed index.scss dist/styles.css",
+    "build": "yarn build-main && yarn build-base && yarn build-utilities && yarn build-clearfix && yarn build-flex && yarn build-position && yarn build-shadows && yarn build-sizing && yarn build-spacing && yarn build-typography && yarn build-visibility",
+    "build-main": "sass --load-path=../../node_modules/ --style=compressed index.scss dist/styles.css",
```

Comment:
Shall we go for the script naming convention with a colon rather than hyphen? This will be consistent with other scripts.